app/assets/javascripts/base.js.erb

```diff
@@ -5,7 +5,6 @@
 //= require utilities/getImageForLink
 //= require honeybadger-js/dist/honeybadger.js
 //= require ahoy.js/dist/ahoy.js
-//= require serviceworker-companion
```

Comment:
This was previously used for Mac/iOS, but is no longer required since the self-destructing service worker does what this file was doing.